app/assets/javascripts/repository_columns/index.js.erb

```diff
@@ -139,21 +151,18 @@
       .find('span.pull-left').text(column.name);
   }
 
-  function _initTagInput() {
+  function _initColumnTypeSelector() {
     $('[name="repository_column[data_type]"]')
       .on('click', function() {
-        var listValueId = 'repository_column_data_type_repositorylistvalue';
-        if($(this).attr('id') === listValueId) {
-          $('[data-role="tagsinput"]').tagsinput({
-            maxChars: <%= Constants::NAME_MAX_LENGTH %>,
-            trimValue: true
-          });
-          $('.bootstrap-tagsinput').show();
-          $('[data-role="tagsimput-label"]').show();
-        } else {
-          $('.bootstrap-tagsinput').hide();
-          $('[data-role="tagsimput-label"]').hide();
-        }
+        $('.column-type').hide();
+        $('[data-column-type="'+$(this).val()+'"]').show();
+    });
+  }
+
+  function _initTagInput() {
+    $('[data-role="tagsinput"]').tagsinput({
+      maxChars: <%= Constants::NAME_MAX_LENGTH %>,
```

Comment:
Clean all ruby code and convert to normal JS file. You can find constans here - constants.js.erb